data/mods/blazemod/items/tools/blob_container.json

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
Wasn't sure whether I should separate these out since they're so few, but here it is. Otherwise unchanged.